packages/synthetic-chain/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "allowImportingTsExtensions": true,
+        "allowSyntheticDefaultImports": true,
+        "checkJs": false, // opt in
+        "maxNodeModuleJsDepth": 2,
+        "module": "NodeNext",
+        "moduleResolution": "nodenext",
+        "noEmit": true,
+        "target": "ESNext"
+    },
+    "include": [
+        "*.js",
+        "*.ts",
+        "proposals/**/*.js",
+        "proposals/**/*.ts",
```

Comment:
These don't seem to be included in `files` in `package.json`. Is that by design?

Reply:
These globs are meant to get all files, independent of what's being published.